application/templates/cronjob.yaml

```diff
@@ -100,6 +98,9 @@ spec:
           {{- with $job.affinity }}
           affinity:
 {{ toYaml . | indent 12 }}
+          {{- end }}
+          {{- with $job.priorityClassName }}
+          priorityClassName: {{ toYaml . | nindent 12 }}
```

Comment:
Did you test that it really works? Now it looks confusing, cause there would be just a single string value and there is no need in `toYaml` rendering with nindent

Reply:
@kseniyashaydurova 
Fixed